docs/building.rst

```diff
@@ -28,8 +28,11 @@ you can usually get help in the ``#libtorrent`` IRC channel on
 	libtorrent has some code in header files, that code will not be
 	compatible with the built library if they see different configurations.
 
-	Always make sure that the same TORRENT_* macros are defined when you
-	link against libtorrent as when you build it.
+	Always make sure that the same TORRENT_* and BOOST_* macros are defined
+	when you link against libtorrent as when you build it. The simplest way
+	to see the full list of macros defined is to build libtorrent with
+	``-n -a`` switches added to b2 command line, which output all compiler
```

Comment:
I think it would make sense to make "b2" -> "``b2``". Or would that be inconsistent with the rest of this file?

Reply:
sure, done